uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/installer/PlayStore.java

```diff
@@ -187,4 +187,49 @@ public void installApp(@NonNull final String searchHint) {
         installAppFromMarketPage();
     }
 
+    public void updateApp(@NonNull final String searchHint) {
```

Comment:
Is this an overriden method? If yes, then let's add override annotation

Reply:
Done